src/wallet/rpc/spend.cpp

```diff
@@ -1300,6 +1300,8 @@ RPCHelpMan sendall()
                         {"lock_unspents", RPCArg::Type::BOOL, RPCArg::Default{false}, "Lock selected unspent outputs"},
                         {"psbt", RPCArg::Type::BOOL,  RPCArg::DefaultHint{"automatic"}, "Always return a PSBT, implies add_to_wallet=false."},
                         {"send_max", RPCArg::Type::BOOL, RPCArg::Default{false}, "When true, only use UTXOs that can pay for their own fees to maximize the output amount. When 'false' (default), no UTXO is left behind. send_max is incompatible with providing specific inputs."},
+                        {"minconf", RPCArg::Type::NUM, RPCArg::Default{1}, "If add_inputs is specified, require inputs with at least this confirmations."},
```

Comment:
I maybe missed it but where the `minconf` is set to 1 inside `sendall`? (default value is 0)

Reply:
The default `minconf` in `sendall` is 1 because `sendall` does not spend unconfirmed inputs.

Reply:
where is that established?

let's say that the user does not pass `minconf`, so the default `min_conf` value inside coin control is 0. So, the `AvailableCoins` call will return outputs with depth 0 as well, which are added to the raw tx.

Reply:
`AvailableCoins` won't return unconfirmed utxos that aren't change unless `include_unsafe` is set to true. However, because this means that `sendall` can still spend unconfirmed change, which means that the default `minconf` is really 0. I have the docs to reflect this.